### PR TITLE
My spiral program done and commited

### LINKED ISSUE #1879: Printing  a spiral matrix using C and C++.
It will take a input and it will print a square array accordingly.
I am working on this program.
spiral

I am attaching an sample output of the program.

### PR DESCRIPTION
**Pull request by harshlohia11**
Please review my PR of issue #1879 
Fixes #1879 
spiralsq

#### Changes proposed in this pull request: 
-A spiral program which will take any number as an output and print the output according to the attached document.
spiralsq_c++

#### Languages Used:  
- C

-C++ 

#### Files Added:
- spiral_square.cpp

- spiral_square_c++.cpp

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by jainaman224** [comment]
@harshlohia11 Complete PR template properly.

File: Spiral_Array/spiral_square.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include<stdio.h>
+int main()
+{
+	int a[10][10],i,j,k=1,t;
```
Comment (jainaman224) [inline Spiral_Array/spiral_square.cpp]: Use 4 space indentation.

File: Spiral_Array/spiral_square.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#include<stdio.h>
```
Comment (jainaman224) [inline Spiral_Array/spiral_square.cpp]: Fix file name.

File: Spiral_Array/spiral_square_c++.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#include<iostream>
```
Comment (jainaman224) [inline Spiral_Array/spiral_square_c++.cpp]: Delete one of the file.

**Comment by jainaman224** [comment]
There are 2 files, `Spiral_Array/spiral_square.cpp` and `Spiral_Array/spiral_square_c++.cpp`. What do you mean by two languages to execute the code?

File: Spiral_Array/spiral_square.cpp (hunk 2 above)
Comment (jainaman224) [reply Spiral_Array/spiral_square.cpp]: Spiral_Array.cpp should be the name of file as well.

**Comment by harshlohia11** [comment]
Sir I meant one file is in C and other is in C++. That was the issue I created to write the program in C and C++.

**Comment by harshlohia11** [comment]
Am working on the changes required.

**Comment by MastersAbh** [comment]
Hi @harshlohia11 , There has been no update from your side for 9 days. Closing your PR for now. If you still want to work on the issue, reach out to us on slack.

**Pull request closed without merging**